packages/design-system/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -7,6 +7,7 @@ import { useArgs } from '@storybook/preview-api';
 const meta: Meta<typeof Accordion> = {
   title: 'Components/Accordion',
   component: Accordion,
+  subcomponents: { AccordionItem },
```

Comment:
I was under the impression that CSF3 doesn't use subcomponents but I do see them showing up in the docs so maybe this was just used in the docs and I removed it not thinking about that. 👍🏻

Reply:
[See the note I left here](https://github.com/CMSgov/design-system/pull/2562/files#diff-569b9b80de2456e5a082d20cc1f491ee06aff354c1d5d930cdfedf2d4193c576R15-R25)

It's technically deprecated, but I don't think they've made a final decision on it yet. For now, the functionality is really handy for us because it means we'll have prop tables for all components involved in a story.